allauth/socialaccount/providers/telegram/provider.py

```diff
@@ -20,8 +20,8 @@ def extract_uid(self, data):
     def extract_common_fields(self, data):
         return {
             "first_name": data["first_name"],
```

Comment:
Shouldn't `first_name` be given the same treatment?

Reply:
first_name is mandatory to have a telegram account and is always present in auth url

Reply:
Do you think the above request needs any improvements?